docs/install-and-run.md

```diff
@@ -214,47 +226,47 @@ compatible with EarlGrey.
 1. **Ctrl+Click** the folder for your app's source files, and then select **New file...**. The
 following dialog will appear:
 
-   <img src="images/image09.png" width="422">
+<img src="images/image09.png" width="422">
 
 2. Select **Unit Test Case Class**, and then click **Next**. On the following screen, type the name
 of your test case. For this example, let’s leave it as *MyFirstEarlGreyTest*:
 
-   <img src="images/image10.png" width="422">
+<img src="images/image10.png" width="422">
 
 3. On the next screen, make sure that the test is associated with the Unit Test target. In this
 case, our target is *SimpleAppTests*:
 
-   <img src="images/image11.png" width="422">
+<img src="images/image11.png" width="422">
 
 4. Xcode will create a new test case for us but we won’t need much of it. Let’s change the code to
```

Comment:
You should have two sections here, one for objc and one for swift. Prefer swift to be on the top